Frontend/src/comopnents/ExpenseCalculator/ExpenseCalculatror.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import "./ExpenseCalculator.css";
+
+let expenseData = [];
+
+const ExpenseCal = () => {
+  return (
+    <div className="container">
+      <div className="heading">
+        <h4>Courier expense</h4>
+      </div>
+      <div className="form">
```

Comment:
You can also use libraries like MUI, Chakra, https://ui.shadcn.com/ instead of writing from scratch.

Reply:
Sorry sir not updating of this many days.
I am facing problem using tailwind css and chakraUI component can you give me some hint how to use tailwind css.

Reply:
Check out the video, 

-> Short to the point: https://www.youtube.com/watch?v=1WdL1TLfkYo 
-> Tailwind CSS Tutorial[2 Hrs]: https://www.youtube.com/watch?v=_9mTJ84uL1Q

if the videos don't help, let's connect over the Google Meet.